mealie/schema/user/user.py

```diff
@@ -66,6 +66,14 @@ class GroupBase(MealieModel):
     class Config:
         orm_mode = True
 
+    @validator("name")
+    def not_null(cls, v: str):
+        v = v.strip()
+        if not v:
+            raise ValueError("name cannot be empty")
+
+        return v
+
```

Comment:
Probably easier to do this with a constrained type

https://docs.pydantic.dev/1.10/usage/types/#constrained-types

tldr:

```python
class Model(BaseModel):
    upper_str: constr(to_upper=True)
    lower_str: constr(to_lower=True)
    short_str: constr(min_length=2, max_length=10)
    regex_str: constr(regex=r'^apple (pie|tart|sandwich)$')
    strip_str: constr(strip_whitespace=True)
```

Reply:
Huh, that's a nice feature, TIL